packages/compiler/lib/decorators.ts

```diff
@@ -480,20 +480,26 @@ export function getTags(program: Program, target: Type): string[] {
 
 // Merge the tags for a operation with the tags that are on the namespace or
 // interface it resides within.
-//
-// TODO: (JC) We'll need to update this for nested namespaces
 export function getAllTags(
   program: Program,
-  container: NamespaceType | InterfaceType,
-  target: Type
+  target: NamespaceType | InterfaceType | OperationType
 ): string[] | undefined {
   const tags = new Set<string>();
 
-  for (const t of getTags(program, container)) {
-    tags.add(t);
-  }
-  for (const t of getTags(program, target)) {
-    tags.add(t);
+  let current: NamespaceType | InterfaceType | OperationType | undefined = target;
+  while (current !== undefined) {
+    for (const t of getTags(program, current)) {
+      tags.add(t);
+    }
+
+    // Move up to the parent
+    if (current.kind === "Operation") {
+      current = current.interface || current.namespace;
```

Comment:
```suggestion
      current = current.interface ?? current.namespace;
```

Reply:
Is there an advantage to using `??` in this case?  My assumption is that they have the same result

Reply:
If it can't be true/false it doesn't really matter, I just think it is better to keep using `??` when you mean if unassigned

Reply:
Fair enough, I'll change it